### PR TITLE
DM-346: Add a what's new section for Advanced Software Management

### PR DESCRIPTION
**Pull request by georgi-huklev**

### CODE REVIEW

File: content/release-10-14-0/whatsnew-10-14-0.md [hunk 1]
```diff
@@ -41,8 +41,33 @@ Additionally, devices and assets can be searched globally (for reports or home d
...
+
+```csv
+140,nvm,0.39.1,apt,https://github.com/nvm-sh/nvm,containerd.io,1.6.0,apt,https://containerd.io/,nginx,1.21.6,container-image,https://hub.docker.com/_/nginx
+```
+
+This would result in the creation of the software artefacts below:
+
+![Advanced Software](/images/release-notes/advanced-software.png)
+
+In order to migrate to the new Software model devices should remove their `c8y_SoftwareList` fragment and set their software artefacts anew using the SmartREST template above.
```
Comment (BeateRixen) [inline content/release-10-14-0/whatsnew-10-14-0.md:58]: Is this described in the documentation how to migrate?

**Review by l2p- — APPROVED** [review]

File: content/release-10-14-0/whatsnew-10-14-0.md (hunk 1 above)
Comment (georgi-huklev) [reply content/release-10-14-0/whatsnew-10-14-0.md:58]: Seems to be missing:
https://github.com/SoftwareAG/c8y-docs/pull/370

Comment (BeateRixen) [reply content/release-10-14-0/whatsnew-10-14-0.md:58]: We should still add a link here to the section where this will be explained.

**Comment by BeateRixen** [comment]
> The entire description only describes the developer perspective. Is there anything to be added fro the UI perspective?

@georgi-huklev How about my question above?

File: content/release-10-14-0/whatsnew-10-14-0.md (hunk 1 above)
Comment (georgi-huklev) [reply content/release-10-14-0/whatsnew-10-14-0.md:58]: @MaciekSoftwareAG can you add that info to your PR?

**Comment by georgi-huklev** [comment]
> The entire description only describes the developer perspective. Is there anything to be added fro the UI perspective?

@l2p- @bozhidardedov what do you guys think about Beate's comment above?

**Comment by bozhidardedov** [comment]
> The entire description only describes the developer perspective. Is there anything to be added fro the UI perspective?

@BeateRixen @georgi-huklev Yes, I will add a paragraph about what's changed in the UI.

**Comment by BeateRixen** [comment]
@georgi-huklev The first image looks very blurry. Could you please replace this with a better one?

**Comment by georgi-huklev** [comment]
> @georgi-huklev The first image looks very blurry. Could you please replace this with a better one?

@BeateRixen I think it has to do with how github display them. Both pictures have the same pixel density and are readable when I open them on my PC.

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-14-0/whatsnew-10-14-0.md (hunk 1 above)
Comment (georgi-huklev) [reply content/release-10-14-0/whatsnew-10-14-0.md]: Oops, I merged it already. I'll create a new PR for that as soon as it is ready.

**Pull request merged**